source/blog/2016-04-29-docker-lvm-plugin.md

```diff
@@ -0,0 +1,67 @@
+ 
+# <span style="color:cornflowerblue">docker lvm plugin</span>
+## <span style="color:cornflowerblue">Author: Shishir Mahajan</span>
+---
+Recently docker introduced a volume command (docker volume –help for more information) to allow users to create a logical volume and then bind mount it inside the container at container creation/run time.  You can achieve this by:
```

Comment:
*Recently* is kind of wrong since `docker volume` is in docker since 1.9.x (we're now about at 1.12)